docs/source/faq/faq.md

```diff
@@ -52,8 +52,8 @@ This is a growing set of technical FAQs. The [product FAQs on the Kedro website]
 
 ## Nodes and pipelines
 
-* [How do I create a modular pipeline](../nodes_and_pipelines/modular_pipelines.md#how-do-i-create-a-modular-pipeline)?
-
+* [How can I create a new blank pipeline using the `kedro pipeline create` command](../nodes_and_pipelines/modular_pipelines.md#how-to-create-a-new-blank-pipeline-using-the-kedro-pipeline-create-command)?
+* [How can I reuse my pipelines with namespaces](../nodes_and_pipelines/namespaces.md)?
```

Comment:
```suggestion
* [How can I reuse my pipelines](../nodes_and_pipelines/namespaces.md)?
```

Reply:
Same here really. If there are multiple ways and you're solving for a particular way, e.g. with namespaces, maybe it is OK, but otherwise I'd avoid making it so long.